apps/pie-docs/src/components/switch/code/props.json

```diff
@@ -2,7 +2,7 @@
     "headings": ["Prop", "Options", "Description", "Default"],
     "rows": [
         [
-            "isChecked",
+            "checked",
```

Comment:
I don't have an issue with changing this to `checked`, but if we're going to do that, likely need to also change `isDisabled` at the same time so that all HTML attribute style props are treated consistently

Reply:
we use `disabled` everywhere else, this is the component that breaks the pattern. Xander and I have agreed to use `disabled` instead but not as part of this work. We can create a new ticket for it :)

Reply:
DSW-1571 😄